### DISCUSSION TITLE
[packaging] failing tests and missing (& unspecified) dependencies

### DISCUSSION THREAD

**Discussion by xgqt** [cyanfish/naps2#447] [Development]
Hello!

All test pass for me on my devbox but on the Gentoo CI test fail with **weird** errors.

Do you have any advice for this bug: https://bugs.gentoo.org/938864
Build log is here: https://938864.bugs.gentoo.org/attachment.cgi?id=901797

Thanks in advance!

**Comment by cyanfish** [comment]
Weird error for sure, I haven't seen it before. It's possible the root cause is a missing `libgdk_pixbuf-2.0.so.0` (i.e. GTK not installed) though I'm not sure.

**Reply by xgqt** [reply]
Pixbuf is installed:

> x11-libs/gdk-pixbuf-2.42.12:2 -abi_mips_n32 -abi_mips_n64 -abi_mips_o32 -abi_s390_32 -abi_s390_64 -abi_x86_32 abi_x86_64 -abi_x86_x32 gif -gtk-doc introspection jpeg -test -tiff

On my system I have "introspection jpeg tiff" (tiff is enabled). I tested with tiff disabled and test still pass. I also tested with exact same version - had not reproduced the issue.

**Reply by cyanfish** [reply]
Yeah, checking without that DLL you do get a DllNotFoundException so that's probably not it. Otherwise it seems like some kind of race condition but I have no idea why it would be happening on the CI box - do you know if it's a consistent failure?

**Reply by xgqt** [reply]
Some pkgs might be experiencing werird behavior with high job count - here it is -maxCpuCount:64 and afaik the box does have those CPU cores.
I will ask on the bug to re-test with 1 (make)job.
